feat: Add purchase page object, locator and feature

MiltonKlun opened the pull request:
This pull request introduces the "Purchase Window" page object, along with adding a locator and it's correspondent feature scenarios:

- purchase.feature: new feature file with scenarios for selecting a product, adding it to the cart, and navigating to the cart page.
- PurchaseWindow.js: new Page Object Model containing methods to perform the actions defined in the feature file.
- locators.js: updated with a new locator for the "Place Order" button on the cart page

emzima24 reviewed (changes requested):
Please correct, in the Purchase Window.js file const locators = Cypress.env("../locators"); Remember that constants are in uppercase LOCATORS.

MiltonKlun commented on cypress/support/pagesObject/PurchaseWindow.js:
Done

MiltonKlun commented:
Made the changes based on your feedback:

- locators.js: renamed the 'cssPlaceOrderButton' to be less verbose than the previous 'cssPlaceOrderButtonOnCart'.
- PurchaseWindow.js: now it inherits from ProductPage to reuse the methods on ProductPage and HomePage.
- purchase.feature: made the steps more descriptive and changed the scenarios to focus on the start of the purchase process, instead of validating if the product exists in the cart.

Duquemore reviewed (approved):
Nice
